pkg/anchor/handler/proof/handler_test.go

```diff
@@ -707,150 +707,71 @@ func (wp *mockWitnessPolicy) Evaluate(_ []*proofapi.WitnessProof) (bool, error)
 
 //nolint:lll
 const anchorEvent = `{
-  "@context": [
-    "https://www.w3.org/ns/activitystreams",
-    "https://w3id.org/activityanchors/v1"
-  ],
-  "index": "hl:uEiDzUEQi2qRreCTfvp2AKmTaxuqUUZZNhbxe5RTBH59AWw",
+  "@context": "https://w3id.org/activityanchors/v1",
```

Comment:
activityanchors does not have the base activitystreams vocabulary.

Reply:
I'll fix it in a separate PR.